pkgs/applications/video/devede/default.nix

```diff
@@ -21,6 +21,11 @@ in buildPythonApplication rec {
 
   buildInputs = [
     ffmpeg
+
+    # Temporary fix
+    # See https://github.com/NixOS/nixpkgs/issues/61578
+    # and https://github.com/NixOS/nixpkgs/issues/56943
+    gobject-introspection
```

Comment:
atm I think we're preferring just disabling `strictDeps`. https://github.com/NixOS/nixpkgs/commit/b12b1e2f42841b47b3d1e2e44078dd5b18f6d78f

Reply:
wait seeing https://github.com/NixOS/nixpkgs/issues/61578#issuecomment-493285373...

Reply:
@worldofpeace unfortunately `strictDeps = false;` did not fix the issue: https://github.com/NixOS/nixpkgs/issues/61578#issuecomment-493285373

Reply:
Checked locally, I didn't work with `strictDeps = false;` only if it's not a `nativeBuildInputs` and didn't check that it wasn't here because I expected it to be.

So adding it to `nativeBuildInputs` and disabling `strictDeps` should work.

Reply:
@worldofpeace Thanks!